### PR TITLE
Complete rip and replace

### PR DESCRIPTION
**Pull request by abossert**
This is to port over the most current version and completely replace an outdated one.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
-assemblyJarName in assembly := name.value + "-" + version.value + ".jar"
-
-resolvers ++= Seq(
-    "clojars"           at "https://clojars.org/repo/"
-)
+credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
+publishMavenStyle := true
+publishTo := {
+  val azureArtifacts =
+    "https://pkgs.dev.azure.com/Punch-Research/2ad94760-8a08-4dea-b480-6ff857d18c06/_packaging/etl-utilities-release/maven/v1"
```
Comment (brbickel) [inline build.sbt]: This should be `https://pkgs.dev.azure.com/Punch-Research/2ad94760-8a08-4dea-b480-6ff857d18c06/_packaging/`, and let the parts below be:

```
  if (isSnapshot.value)
    Some("snapshots" at azureArtifacts + "etl-utilities-release/maven/v1")
  else
    Some("releases" at azureArtifacts + "etl-utilities-snapshot/maven/v1")
```

You could create a set of artifact repos for kafka-serde, but it probably makes sense to keep them at the project name level.

File: .scalafmt.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+version = 2.7.5
```
Comment (brbickel) [inline .scalafmt.conf:1]: I'm not familiar with this file type. Was it put here by the IDE?

**Review by brbickel — CHANGES_REQUESTED** [review]
@abossert check out the comments. At a minimum, the `build.sbt` needs to be updated because it will use an invalid URL.

File: .scalafmt.conf (hunk 2 above)
Comment (abossert) [reply .scalafmt.conf:1]: no, it is the code style plugin for Scala....it is the most commonly used one and needs a config file

**Review by brbickel — APPROVED** [review]

**Pull request merged**